### ISSUE TITLE
Docs broken about branch

### ISSUE THREAD

**Issue by nemocurcic** [acdlite/recompose#287]
The docs contains this:
```js
// `hasLoaded()` is a function that returns whether or not the component
// has all the props it needs
const spinnerWhileLoading = hasLoaded =>
  branch(
    props => !hasLoaded(props),
    renderComponent(Spinner) // `Spinner` is a React component
  )

// Now use the `spinnerWhileLoading()` helper to add a loading spinner to any
// base component
const enhance = spinnerWhileLoading(
  props => props.title && props.author && props.content
)
const Post = enhance(({ title, author, content }) =>
  <article>
    <h1>{title}</h1>
    <h2>By {author.name}</h2>
    <div>{content}</div>
  </article>
)
```

But `branch` doesn't support only 2 arguments. I saw that there were a PR about this but it's not merged. https://github.com/acdlite/recompose/pull/227

Please publish the new functionality or change the docs!

**Comment by istarkov** [comment]
Merged, 
![image](https://cloud.githubusercontent.com/assets/5077042/21113298/09df0a9a-c0ba-11e6-9e48-5f07c98c2856.png)
not published yet.

Documentation updated according to PRs, so the problem in releases,
having that only @acdlite can publish this repo, I can't say much when, why, who.

**Comment by istarkov** [comment]
PS: Until release just use latest tag for the documentation
https://github.com/acdlite/recompose/tree/v0.20.2

**Comment by istarkov** [comment]
Oops, 
now I can publish on npm, 
so new version available on npm 0.21.0, please check.

;-)
